pages/accountLists/[accountListId].page.test.tsx

```diff
@@ -108,4 +108,91 @@ describe('AccountListsId page', () => {
       expect(getByText('Good Morning, firstName.')).toBeInTheDocument();
     });
   });
+  describe('AccountListId Redirects', () => {
+    const restApiNotFoundErrorMessage = "Resource 'AccountList' is not valid";
+
+    it('replaces and redirects to the default account list id if Not Found', async () => {
+      const query = jest.fn();
+
+      (makeSsrClient as jest.Mock).mockReturnValue({
+        query: query,
+      });
+      query.mockRejectedValueOnce(new Error(restApiNotFoundErrorMessage));
```

Comment:
Try making this an `ApolloError`. That should make those uncovered lines be covered.